documentation/suppression-examples.md

```diff
@@ -0,0 +1,38 @@
+# Suppression Examples
```

Comment:
is adding this file as part of this PR intentional?

Reply:
I'm going to move the file to another branch.